styles/index.css

```diff
@@ -1,70 +1,110 @@
 * {
   box-sizing: border-box;
 }
+
+/*--elements--*/
+
 body {
-  margin: 0 0 0 0;
+  margin: 0;
   display: grid;
   grid-template-rows: 60px auto 60px;
   height: 100vh;
 }
-main {
-  background-color: black;
-  overflow-y: auto;
-  display: grid;
-  grid-gap: 20px;
-  height: 100%;
-}
-header {
-  justify-content: center;
-  display: grid;
-}
 
 h1 {
-  justify-content: center;
+  text-transform: uppercase;
+  text-align: center;
   background-color: yellow;
   padding: 10px;
   margin: 0;
   border-bottom: 6px solid black;
+  width: 100%;
 }
 
-.body {
+main {
+  background-color: black;
+  overflow-y: auto;
+  display: grid;
+  grid-gap: 16px;
+  padding: 10px 16px 16px;
 }
 
-.card-set {
-  background-color: yellow;
-  padding-left: 1em;
-  padding-right: 1em;
+nav {
+  height: 64px;
   display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  border-top: 6px solid black;
+  background-color: yellow;
+}
+
+/*--components--*/
+
+.main__card {
+  background-color: yellow;
+  padding: 0 16px 0;
+  display: flex;
   flex-direction: column;
   border-radius: 8px;
 }
 
-.button {
+.card__button {
   justify-content: center;
   background-color: black;
   color: yellow;
   padding: 10px 40px;
   text-transform: uppercase;
   border-radius: 8px;
   border: none;
-  display: block;
   align-self: center;
 }
 
-.ul-card {
+.card__tag-set {
   list-style: none;
   display: flex;
   justify-content: center;
 }
 
-.tag {
-  padding: 3px;
-  padding: 0.3em;
+.card__tag {
+  padding: 4px;
   border-radius: 10px;
   margin: 4px;
   background-color: black;
   color: yellow;
 }
 
-.bookmarks {
+/*--components--*/
+
+.app__navigation ul,
+.app__navigation li,
+.app__navigation a {
+  display: contents;
 }
+
+.nav__section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__section--active {
+  background-color: black;
+  color: yellow;
+}
+
+.nav__section > a {
```

Comment:
both properties can be given to nav__section, the children inhibit them automatically.

Reply:
@F-Kirchhoff Not in this case as anchor elements have their own default value.